woocommerce/woocommerce-ios
# Custom help content for "Enter Store credentials" screen

Part of: #7547 

### Description
As a part of improving the help center content, this PR adds custom content with FAQs and Answers for the "Enter Store credentials" screen at the following URL.

https://woocommerce.com/document/android-ios-apps-login-help-faq/#enter-store-credentials

Internal ref - pe5sF9-sZ-p2

### Testing instructions

1. Install and launch the app.
1. Tap `Skip` to reach the login prologue screen.
1. Tap the `Enter Your Store Address` button.
1. Enter a valid store address and tap the `Continue` button
1. Tap the `Sign in with store credentials` button.
1. Tap `Help` -> `Help Center`
1. Observe that you are directed to https://woocommerce.com/document/android-ios-apps-login-help-faq/#enter-store-credentials in a web view
1. In Xcode debug console observe that the following event is tracked 
```
Tracked support_help_center_viewed, properties: [AnyHashable("source_flow"): "login_site_address", AnyHashable("help_content_url"): "https://woocommerce.com/document/android-ios-apps-login-help-faq/#enter-store-credentials", AnyHashable("source_step"): "username_password"]
```

### Screenshots

https://user-images.githubusercontent.com/524475/187134028-215aee2e-cc64-4b49-8072-b707c039a85e.mov



---
